### ISSUE TITLE
LDAP users not deleted when sync

### ISSUE THREAD

**Issue by plmorpheus** [RocketChat/Rocket.Chat#19012]
### Description:

Users deleted from AD/LDAP are not automatically deleted on sync and need to be removed manually.

### Steps to reproduce:

1. Delete user under in AD/LDAP
2. Perform sync in RC

### Expected behavior:

Users previously deleted in AD/LDAP should be automatically deleted in RC when their accounts are not found during sync.

### Actual behavior:

Users remain in RC's database and need to be deleted automatically.

### Server Setup Information:

Version of Rocket.Chat Server: 3.6.2
Operating System: Ubuntu
Deployment Method: snap
Number of Running Instances: 1
DB Replicaset Oplog:
NodeJS Version: 12.16.1
MongoDB Version: 3.6.14

### Client Setup Information

Desktop App or Browser Version: Chrome 85.0.4183.121 / Windows Desktop App ver. 2.17.9
Operating System: Windows

**Comment by ankar84** [comment]
I don't think that delete user in Rocket Chat in such way a best solution. Deactivate is more careful decision.

**Comment by bellegarde-c** [comment]
Deleting users in Universities is a real need. We have many new users every years and many just leave.
